samples/interstitialSample/src/main/java/com/inmobi/interstitial/sample/InterstitialAdsActivity.java

```diff
@@ -116,9 +118,9 @@ public void onAdLoadFailed(InMobiInterstitial inMobiInterstitial, InMobiAdReques
                     }
 
                     @Override
-                    public void onAdReceived(InMobiInterstitial inMobiInterstitial) {
-                        super.onAdReceived(inMobiInterstitial);
-                        Log.d(TAG, "onAdReceived");
+                    public void onAdFetchSuccessful(@NonNull InMobiInterstitial inMobiInterstitial, @NonNull AdMetaInfo adMetaInfo) {
+                        super.onAdFetchSuccessful(inMobiInterstitial, adMetaInfo);
+                        Log.d(TAG, "onAdFetchSuccessful");
```

Comment:
optional  - please show case how to get bid from metaInfo

Reply:
Will include this in CustomAB Sample